.prettierrc

```diff
@@ -1,9 +1,9 @@
 {
   "arrowParens": "always",
   "bracketSpacing": true,
-  "endOfLine": "lf",
+  "endOfLine": "auto",
   "insertPragma": false,
-  "printWidth": 80,
+  "printWidth": 120,
```

Comment:
Why?

Reply:
Personal preference, to be honest. On today's screens which are mostly widescreen, just 80 feels a bit limiting. I've been working with 120 in most projects for years now :)

But also here: If you prefer 80, I'll gladly revert back to it.

Reply:
Just updated the PR with `printWidth: 80` again.

Reply:
Well, I'm not that opposed to large line limits; I have some myself. Esp. for diff viewing a lower line length limit is more comfortable though.